src/transformers/modeling_utils.py

```diff
@@ -866,9 +866,11 @@ def _load_state_dict_into_meta_model(
         module_name = param_name
         set_module_kwargs = {}
 
+        is_param_float8_e4m3fn = hasattr(torch, "float8_e4m3fn") and param.dtype == torch.float8_e4m3fn
```

Comment:
let's do this outside the for loop it will slow things down

Reply:
you mean the hasattr(torch, "float8_e4m3fn") check ? bc i need to check the dtype of the param afterwards

Reply:
I changed a bit, lmk if this is better